### ISSUE TITLE
`jwt_vp_json` support?

### ISSUE THREAD

**Issue by yhuard** [Sphereon-Opensource/pex-openapi#34]
Hi!
I see that you support `jwt_vc_json` but not `jwt_vp_json`, is it on purpose?
In the PEX library, `jwt_vp_json` is used here https://github.com/Sphereon-Opensource/PEX/blob/d9b5c9e9272bbf1d9c96a5188ccefbcc7ea79c99/lib/evaluation/evaluationClientWrapper.ts#L452C36-L452C47

**Comment by yhuard** [comment]
Actually, in the PEX library, `jwt_vp_json` is not supported in the Presentation Definition either https://github.com/Sphereon-Opensource/PEX/blob/d9b5c9e9272bbf1d9c96a5188ccefbcc7ea79c99/resources/presentation_definition_v2.schema.json#L42

**Comment by nklomp** [comment]
No is not on purpose. It should indeed be in there in both places

**Comment by yhuard** [comment]
Thank you @nklomp, I was not sure if I was misinterpreting the specs or not.

It's a bit disturbing to have the [OpenID4VP specs](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-w3c-verifiable-credentials) mention:

> The Credential format identifiers are `jwt_vc_json` for a W3C Verifiable Credential and `jwt_vp_json` for W3C Verifiable Presentation.

while the [PEX v2 specs](https://identity.foundation/presentation-exchange/spec/v2.0.0/#claim-format-designations) relies on the [Claim Format Registry](https://identity.foundation/claim-format-registry/) which does not define `jwt_vc_json` nor `jwt_vp_json`.

**Comment by nklomp** [comment]
Yeah agree. That is also one of the reasons this went wrong probably. The OID4VP spec added new formats not in the registry.
PEX-models has been updated (2.2.2). New PEX version will follow shortly
